src/oscar/apps/catalogue/managers.py

```diff
@@ -18,7 +18,16 @@ def base_queryset(self):
 
     def browsable(self):
         """
-        Excludes non-canonical products.
+        Excludes non-canonical products and disabled products
```

Comment:
For consistency of documentation please change this to "and non-public products". Disabled does not mean the same as "not public".

Reply:
feedback applied